homeassistant/helpers/entity_component.py

```diff
@@ -229,15 +229,17 @@ def async_update_group(self):
 
         This method must be run in the event loop.
         """
+        def sorted_ids():
```

Comment:
Right, that key extraction ended up a bit redundant. I inlined the short version now.